doc/engineering-reference/src/simulation-models-encyclopedic-reference-005/occupant-thermal-comfort.tex

```diff
@@ -336,163 +364,224 @@ \subsubsection{Pierce Two-Node Model Nomenclature List}\label{pierce-two-node-mo
+Or as shown in the EnergyPlus code:
 
-\begin{itemize}
-\tightlist
-\item
-  Hr*RadTemp))/(CloCond + CloBodyRat*(Hc + Hr))
-\end{itemize}
+CloSurfTemp = (CloCond*SkinTemp + CloBodyRat*(Hc*AirTemp+Hr*RadTemp))
 
-Total evaporative heat loss from the skin, E\(_{sk}\), includes evaporation of water produced by regulatory sweating, E\(_{rsw}\), and evaporation of water vapor that diffuses through the skin surface, E\(_{diff}\).
+~~~~~~~~~~~~~~~~~~~~~~~/(CloCond + CloBodyRat*(Hc+Hr))
 
-\({E_{sk}} = {E_{rsw}} + {E_{diff}}\) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ W/m\(^{2}\)
+The total evaporative heat loss from the skin, E\(_{sk}\), includes evaporation of water produced by regulatory sweating, E\(_{rsw}\), and evaporation of water vapor that diffuses through the skin surface, E\(_{diff}\).
 
-EvapHeatLoss = EvapHeatLossRegSweat + EvapHeatLossRegDiff
+\begin{equation}
+{E_{sk}} = {E_{rsw}} + {E_{diff}}
+\end{equation}
+
+\begin{equation}
+{E_{rsw}} = 0.68 \times S{W_{reg}}
+\end{equation}
+
+\begin{equation}
+{E_{diff}} = {w_{diff}} \times {E_{\max }}
+\end{equation}
 
-\({E_{rsw}} = 0.68 \times S{W_{reg}}\) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ W/m\(^{2}\)
+Within the EnergyPlus source code, these equations are shown as:
 
-\({E_{diff}} = {w_{diff}} \times {E_{\max }}\) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ W/m\(^{2}\)
+EvapHeatLoss = EvapHeatLossRegSweat + EvapHeatLossRegDiff
 
 RegHeatLoss = 0.68*RegSweat
 
 DiffHeatLoss = SkinWetDiff*MaxEvapHeatLoss
 
-Where,
-
-0.68 is the passive water vapor diffusion rate in g/h·m\(^{2}\)·Torr
-
-and,
+where 0.68 is the passive water vapor diffusion rate in g/h·m\(^{2}\)·Torr and
 
 \begin{equation}
 {w_{diff}} = 0.06(1 - {w_{rsw}})
 \end{equation}
 
-\({E_{\max }} = {h_e}'({P_{sk}} - {P_a})\) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ W/m\(^{2}\)
+\begin{equation}
+{E_{\max }} = {h_e}'({P_{sk}} - {P_a})
+\end{equation}
 
 \begin{equation}
 {w_{rsw}} = {E_{rsw}}/{E_{\max }}
 \end{equation}
 
-SkinWetDiff = (1.-SkinWetSweat)*.06
+These equations within the EnergyPlus source code appear as:
+
+SkinWetDiff = .06*(1.-SkinWetSweat)
 
 MaxEvapHeatLoss = (1./TotEvapHeatResist)*(SatSkinVapPress - VapPress)
 
 SkinWetSweat = EvapHeatLossRegSweat/MaxEvapHeatLoss
 
 The Pierce model has one additional heat flow term describing the heat transfer between the internal core compartment and the outer skin shell (Doherty and Arens 1988).
 
-\({Q_{crsk}} = (5.28 + 1.163SKBF)({T_{cr}} - {T_{sk}})\) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ W/m\(^{2}\)
+\begin{equation}
+{Q_{crsk}} = (5.28 + 1.163SKBF)({T_{cr}} - {T_{sk}})
+\end{equation}
+
+or as shown in the EnergyPlus source code:
 
 HeatFlow = (CoreTemp-SkinTemp)*(5.28 + 1.163*SkinBloodFlow)
 
-Where
+where 5.28 is the average body tissue conductance in W/m\(^{2}\)•°C and 1.163 is the thermal capacity of blood in W•h/L•°C
```

Comment:
Looks like some of the unicode characters may have gotten scrambled up. Maybe:

`W/m\(^2 \mdot \) °C`

Reply:
Sorry that should be:

`W/m\(^2 \cdot \) °C`

I've been working with mass flow rates too much. `\cdot` is center-dot. You could also use `\bullet` if you wanted to match the original Word closer.

Reply:
@michael-okeefe Thanks for the review--I'll fix that as soon as possible.

Reply:
@michael-okeefe By the way, I have been building the PDF locally on my computer to make sure that things are formatted correctly.  So, hopefully once it's up in GitHub things should be good to go other than any typos, etc.

Reply:
Awesome. Sounds great!

Reply:
@RKStrand This still needs to be addressed but once that is fixed, this can be merged in.

Reply:
@Myoldmopar Yes, I will correct this very soon along with equation you mentioned and get this checked back in.  Will keep you posted...

Reply:
@michael-okeefe @Myoldmopar Ok, this issue is fixed now.  Changed the degrees C to K...because despite many attempts I couldn't be both of the degree symbols to show up.  Weird but using just K works and is the same thing.

Reply:
Yeah, LaTeX is definitely a black art. In the past, I've used:

`$^{o}$C` or `$^{\circ}$C` to get the "degree" symbol to print although there are more eloquent ways. `K` is fine with me although I'll defer to Edwin.

Great work!